README.md

```diff
@@ -4,7 +4,20 @@
 [![Chat](https://img.shields.io/static/v1.svg?label=chat&message=mattermost&color=145dbf)](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind)
 
 # Codewind for VS Code
-Create and develop cloud-native, containerized web applications from VS Code.
+Codewind for VS Code provides tools to help build cloud-native, containerized web applications from VS Code, regardless of which IDE or language you use. Rapidly create an application from a template project and launch, update, test, and debug in Docker containers on the desktop.
```

Comment:
I would say "containerized applications" as they aren't always just web.  Also because they would be viewing this from the VS Code marketplace, we may be able to eliminate the "IDE or" and just say regardless of which language you use.